panoptes_aggregation/extractors/utilities.py

```diff
@@ -11,3 +12,57 @@ def annotation_by_task(classification_in):
         ann_by_task[annotation['task']].append(annotation)
     classification['annotations'] = dict(ann_by_task)
     return classification
+
+
+def pluck_fields(classification, pluck_keys):
+    '''
+    Main function that is called by the pluck_extractor.
+    Functions similarly to the PluckFieldExtractor
+    on Caesar, where given a mapping between a key and
+    location of the data on the classification JSON,
+    this extractor returns a dictionary with each extracted
+    value given by the corresponding key.
+
+    Input
+    -----
+    classification : dict
+        A dictionary containing an `annotations` key that is a list of
+        panoptes annotations
+
+    pluck_keys : string
+        String with a dictionary-like formatting that maps the output key to
+        the location of the value in the classification dict. Must begin
+        with a `{` and end with `}`, with each key map separated by a `,`.
+        Each entry should be in the form 'key':'location' (see examples below).
+
+
+    Examples
+    --------
+    >>> classification = {"id": 359841171, "subject": { \
+            "id": 68561182, \
+            "metadata": { \
+              "filename": "uds_H.gf19767.png", \
+              "uber_flag_digit": "4", \
+              "is_gold_standard": "False" \
+            }, \
+        }}
+    >>> pluck_keys = '{"gold_standard":"subject.metadata.is_gold_standard", "true_value":"subject.metadata.uber_flag_digit"}'
+    >>> pluck_fields(classification, pluck_keys)
+    defaultdict(<class 'list'>, {'pluck.gold_standard': 'False', 'pluck.true_value': '4'})
+    '''
+    answers = {}
+
+    pluck_key_dict = ast.literal_eval(pluck_keys)
```

Comment:
To bring this more in line with the other extractor code (e.g. subtask details) can this evaluation be moved to the `extractor_wrapper` on line 27 and this code changed to take in a dictionary object rather than a string formatted like a dictionary?

When not sending the object of a URL it is much easier to just send in a dictionary object (e.g. the configuration yaml files create python dictionaries anyways, might as well keep it in that format), the only time it needs conversion from a string is if submitted via URL and that is exactly what the `extractor_wrapper` is designed to handle.